modules/client/variables.tf

```diff
@@ -54,11 +54,6 @@ variable "client_region" {
   default     = "us-central1"
 }
 
-variable "storage_bucket_location" {
-  description = "GCS storage bucket location"
-  default     = "us-central1"
-}
-
 variable "network" {
```

Comment:
The `network_project` variable does not have a default variable like it does in the server `variables.tf` file. Can we add `default = ""` here, or is that not needed?

Reply:
I had thought about that too.   It looks like the pattern is to leave this without a default and let it be resolved in the "locals" block in main.tf.
https://github.com/kevensen/terraform-google-forseti/blob/17edae8394a1af8b4a1a74dc86daa349615e6f24/modules/client/main.tf#L33

Reply:
uh-oh.  I misunderstood what you were asking.  Yes.  Will fix.